projects/dapr/cert-manager-graduation.md

```diff
@@ -0,0 +1,501 @@
+
+- [x] **Document what the project does, and why it does it - including viable cloud native use cases.**
+
+  Dapr reduces the complexity of interacting with cloud infrastructure such as databases, message brokers, configuration stores, secret stores etc. by providing developers with a consistent API that can target multiple infrastructure backends (database, message broker etc.). Developers use Dapr to tackle hard distributed systems challenges like durable workflows, business orchestration, state conflict resolution etc. 
+  
+  An additional goal of the project is to secure applications at the application level instead of relying on network or infrastructure level security. Dapr allows platform teams to scope access to infrastructure to specific applications. And when integrating with a database, Dapr automatically creates data isolation per service among others. 
+  
+  Dapr also includes native authentication and authorization middleware for developers to use.
+  
+  Although Dapr utilizes sidecars, the project has specified that it is not a [service mesh replacement](https://v1-13-1.docs.dapr.io/concepts/service-mesh/). According to Adopter interviews, there is still confusion on where Dapr fits in with Service Mesh technologies.
+
+- [x] **Document and maintain a public roadmap or other forward looking planning document or tracking mechanism.**
+
+  The Dapr project has a documented release process that includes a 2 week “feature definition stage” in which maintainers triage items based on community feedback into the milestone. This determines the short term roadmap for a given release. For long term items they have a dedicated proposal feature where big ticket items are opened by the community and maintainers, and are then discussed and voted on. Once an issue is approved, it will roll into a specific release during the feature definition stage.
+
+  The [Dapr high-level roadmap](https://github.com/dapr/community/blob/master/roadmap.md) is maintained in the Dapr community repository and is updated quarterly. The current update cadence is not frequent enough, allowing for out of sync information - [tracking issue #4377](https://github.com/dapr/docs/issues/4377) in the Dapr docs repo has been created to track updates to this documentation. The current roadmap documentation also does not sufficiently reflect the current process and is being tracked by the same issue.
+
+- [ ] **Roadmap change process is documented.**
+
+  There is no current roadmap change process documented. [Tracking issue #4377](https://github.com/dapr/docs/issues/4377) in the Dapr docs repo has been created to track Roadmap process improvements including creating and documenting a roadmap change process.
```

Comment:
Has this been corrected? If so, do the tracking issue contain sufficient information that this Criteria is now met and no longer blocking?

Reply:
Updated with latest commit - the roadmap change process has been sufficiently documented to meet this Graduation requirement.